modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/variables/_buttons.scss

```diff
@@ -123,3 +123,15 @@ $btn-link: (
 		color: var(--color-action-primary-disabled),
 	),
 );
+
+$btn-colors: (
```

Comment:
Adding extra variables makes managing variables harder. It's better to repeat it for each variant.